### ISSUE TITLE
Add proper icon

### ISSUE THREAD

**Issue by te-online** [te-online/nextcloud--files_linkeditor#3]

**Comment by DJCrashdummy** [comment]
_just my 2 cents:_
at least for now the globe used as file-icon is quite ok... it just needs to be added as app-icon.

**Comment by Shen** [comment]
Currently there is shown a gearwheel if I add a link. Is it possible to change this to a globe- or link-symbol?

**Comment by te-online** [comment]
In the appstore, you mean? Yeah, it could be nice to have an SVG globe there. Would you mind looking for the icon in github.com/nextcloud/server and then create a PR to add the icon so the appstore can read it? (For inspiration I'd look at how other apps do it). 😊

**Comment by Shen** [comment]
No, if I create a new Link in Files-App.
![8851A25F-91E9-456D-A455-BDC006191483](https://user-images.githubusercontent.com/81589/68549917-19f0a800-03fe-11ea-8b8d-9a5a2f291323.jpeg)

**Comment by te-online** [comment]
@Shen Is this screenshot taken from the web interface of Nextcloud? I can't reproduce the wrong icon in version 16 or 17 🤔 For me it shows the globe.
![Bildschirmfoto 2019-11-11 um 21 26 27](https://user-images.githubusercontent.com/4127167/68618582-f1ce7b00-04c9-11ea-84fb-fb22f5f1de9d.png)

**Comment by Shen** [comment]
Yes, it is from the web interface of Nextcloud (16.0.5.1). Strange...

**Comment by te-online** [comment]
Sorry that I can't say what's wrong... 

Maybe you can right-click the icon and click “Inspect element“, then paste a screenshot of that HTML node here?

I'm guessing that maybe another app is interfering or something like that?

**Comment by Shen** [comment]
Something like this?
![linkeditor-icon-bug](https://user-images.githubusercontent.com/81589/72101281-a4815500-3324-11ea-9304-1e7c99a2ed75.png)

**Comment by te-online** [comment]
I have a suspicion that this has to do with the “Theming” app. Let me look into this once I get a bit of time... Sorry for the delay!

**Comment by te-online** [comment]
@Shen Can you check if you even have the correct file under `/index.php/apps/theming/img/core/filetypes/link.svg`?
